api/wing/schema.py

```diff
@@ -26,20 +27,29 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, **kwargs):
-        validate_empty_fields(**kwargs)
-        floor = Floor.query.filter_by(id=kwargs['floor_id']).first()
-        if not floor:
-            raise GraphQLError("Floor not found")
-        admin_roles.check_office_location_create_wing(floor_id=kwargs['floor_id'])  # noqa: E501
-        admin_roles.create_update_delete_wing()
-        wing = WingModel(**kwargs)
-        payload = {
-            'model': WingModel, 'field': 'name', 'value':  kwargs['name']
-        }
-        with SaveContextManager(
-          wing, 'Wing', payload
-        ):
-            return CreateWing(wing=wing)
+        try:
+            validate_empty_fields(**kwargs)
+            floor = Floor.query.filter_by(id=kwargs['floor_id']).first()
+            if not floor:
+                raise GraphQLError("Floor not found")
+            admin_roles.check_office_location_create_wing(floor_id=kwargs['floor_id'])  # noqa: E501
+            admin_roles.create_update_delete_wing()
+            wing = WingModel(**kwargs)
+            payload = {
+                'model': WingModel, 'field': 'name', 'value':  kwargs['name']
+            }
+            query = Wing.get_query(info)
+            result = query.filter(WingModel.state == "active")
```

Comment:
kindly remove this code because its redundant with the presence of the context manager